README.md

```diff
@@ -35,7 +35,7 @@ Yarn is a javascript package management for all dependencies. It will be used to
 It's probably best to have two terminal tabs open, as you will need to build and transpile your React components; and then Jekyll will serve it to your local host.
 
 #### Building the React Components
-To build and tranpile the React Components you will need to run the command `webpack` in your command line
+To build and transpile the React Components you will need to run the command `npm run build` in your command line
```

Comment:
Should we change this instruction to `yarn run build`? I think we're moving towards yarn where we can?